src/app/View.ts

```diff
@@ -0,0 +1,115 @@
+import { Render } from './utils/Render';
+import { Timer } from './Timer';
+import { AnimalNames } from '../Enums/AnimalNamesEnum';
+import { Player } from '../Player';
+
+export class View {
+  renderMenuView(): void {
+    const inputName = Render.elementFactory('input', {
+      className: 'input__name',
+      placeholder: 'Carlos Santana',
+    });
+    const startGameButton = Render.elementFactory(
+      'button',
+      {
+        className: 'button__start button',
+      },
+      'Start Game',
+    );
+    const avatarsElements = Object.values(AnimalNames).map(
+      (animalName) => {
+        if (animalName === AnimalNames.BIG_DOG) {
+          return Render.elementFactory('img', {
+            src: `./static/images/avatars/dog.png`,
+          });
+        }
+        return Render.elementFactory('img', {
+          src: `./static/images/avatars/${animalName}.png`,
+        });
+      },
+    );
+    Render.render(
+      '#sf-app',
+      inputName,
+      startGameButton,
+      ...avatarsElements,
+    );
+
+    let playersChosenAvatarPath = '';
+    avatarsElements.forEach((el) => {
+      el.addEventListener('click', (e): void => {
+        playersChosenAvatarPath = (e.target as any).src;
+        console.log(playersChosenAvatarPath);
+      });
+    });
+
+    const handleClick = () => {
+      const inputValue =
+        (<HTMLInputElement>inputName).value === ''
+          ? 'Carlos Santana'
+          : (<HTMLInputElement>inputName).value;
+      const newPlayer = new Player(
```

Comment:
Player should be created in GameController, but for now you can mark it to refactor

Reply:
I can use that actually with my solution (for now).